tensorflow/python/framework/ops_test.py

```diff
@@ -254,6 +254,10 @@ def testInputsAndOutputs(self):
     input:'myop1' input:'myop2:1' input:'myop2:1'
     """, op3.node_def)
 
+  def testDevicePresent(self):
+    op = ops.Operation(ops._NodeDef("None", "myop", device='/job:goo/device:GPU:0'), ops.Graph(), [], [])
```

Comment:
Drive-by comment: Please limit column length to 80. Follow the examples of other parts of the file for how to break lines and indent properly.

Reply:
Done!